### PR TITLE
Scientific works

### PR DESCRIPTION
**Pull request by Kwolik**
- Add component: 'Scientific works'
- Create 4 files:
   a) Search.jsx
   b) ScientificWorksCategories.jsx
   c) ScientificWorksOneWork.jsx
   d) ScientificWorksRecentAuthors.jsx

obraz

### CODE REVIEW

**Comment by upsu90** [comment]
remember not to use negative margins :p

**Review by JustD4nTe — APPROVED** [review]

File: web-client/src/components/ScientificWorksOneWork.jsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    //https://www.xspdf.com/resolution/50694881.html <- informacje do buttona download
+    return(
+        <div className={style.main}>
+            <h2 className={style.h2}>{props.title}</h2>
+            <div className={style.panel}>
+                <a href={props.link}><p className={style.category}>{props.categories}</p></a>
+                <p className={style.dot}>.</p>
+                <p className={style.date}>{props.data}</p>
+                <p className={style.dot}>.</p>
```
Comment (upsu90) [inline web-client/src/components/ScientificWorksOneWork.jsx]: use symbols instead

File: web-client/src/pages/scientific-works.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+            width: '65%', 
+            float: 'left',
+        },
+        right:
+        {
+            paddingTop: '5%',
+            width: '35%',
+            float: 'right',
+        },
+        h4:
```
Comment (upsu90) [inline web-client/src/pages/scientific-works.js]: you can use h3

**Review by upsu90 — CHANGES_REQUESTED** [review]

**Review by KamilDonda — APPROVED** [review]

**Review by upsu90 — APPROVED** [review]
ok:)

**Pull request merged**